inst/fill/fill.css

```
@@ -18,4 +18,10 @@
     /* Prevent shrinking or growing of non-fill items */
     flex: 0 0 auto;
   }
+  .html-fill-container > .shiny-html-output:empty,
+  .html-fill-container > .shiny-text-output:empty {
```

Instead of targeting shiny output classes specifically, we could generically target empty divs

```suggestion
  .html-fill-container > div:empty {
```